lib/fs.js

```diff
@@ -134,6 +134,19 @@ function maybeCallback(cb) {
   throw new ERR_INVALID_CALLBACK();
 }
 
+function isBuffersArray(value) {
+  if (!Array.isArray(value) || value.length === 0)
```

Comment:
```suggestion
  if (!Array.isArray(value))
```

I don’t think we would want to explicitly forbid `value.length === 0`?

Reply:
I do not understand why we would accept an empty array as an input?

Reply:
@AnasAboureada Because we accept an arbitrary number of buffers as input, and arbitrary should include zero, because imo that’s the less surprising behaviour.